src/tiled_node/tiled_node.cpp

```diff
@@ -384,25 +378,45 @@ void HexMap::set_cell_item(const HexMapCellId &cell_id,
         updated_cells.insert(cell_key);
 
         ERR_FAIL_COND_MSG(octant == nullptr, "octant for cell does not exist");
-        octant->remove_cell(cell_key);
+        octant->clear_cell(cell_key);
         update_dirty_octants();
     }
 
     return;
 }
 
-void HexMap::_set_cell_item(const Ref<hex_bind::HexMapCellId> cell_id,
+void HexMapTiledNode::_set_cell_item(const Ref<hex_bind::HexMapCellId> cell_id,
         int p_item,
         int p_rot) {
     ERR_FAIL_COND_MSG(!cell_id.is_valid(), "null cell id");
     set_cell_item(**cell_id, p_item, p_rot);
 }
 
-void HexMap::_set_cell_item_v(const Vector3i &cell_id, int p_item, int p_rot) {
+void HexMapTiledNode::_set_cell_item_v(const Vector3i &cell_id,
+        int p_item,
+        int p_rot) {
     set_cell_item(cell_id, p_item, p_rot);
 }
 
-int HexMap::get_cell_item(const HexMapCellId &cell_id) const {
+Array HexMapTiledNode::get_cell_ids_v() const {
+    Array out;
+    for (const auto &iter : cell_map) {
+        out.push_back((Vector3i)iter.key);
+    }
+    return out;
+}
```

Comment:
Hmm, HexMapNode should provide some method to get the array of defined cells.  If not, implement it and remove this one.  Although, we'll probably need to keep this as the subclass impl is different for intnode.